code/parse/sexp.cpp

```diff
@@ -14668,7 +14669,26 @@ void sexp_ship_create(int n)
 		vm_angles_2_matrix(&new_ship_ori, &new_ship_ang);
 	}
 
-	ship_create(&new_ship_ori, &new_ship_pos, new_ship_class, new_ship_name);
+	int objnum = ship_create(&new_ship_ori, &new_ship_pos, new_ship_class, new_ship_name);
+	Assert(objnum != -1);
+
+	// do some initialization that is usually handled by parse_create_object_sub
+	int shipnum = Objects[objnum].instance;
+	ship *shipp = &Ships[shipnum];
+	ship_info *sip = &Ship_info[shipp->ship_info_index];
+
+	ship_assign_sound(shipp);
```

Comment:
`ship_create` simply creates an instance of a ship class, while `parse_create_object` customizes it according to whatever the FREDder configured on a per-ship basis.  Warp effects and no-shields can be configured differently for otherwise identical ships, so they can't be in `ship_create`.  It does appear, however, that `ship_assign_sound` could be moved inside `ship_create`.

Reply:
cool - thanks for the explanation!